pkg/monitoring/metrics/virt-controller/vmsnapshot.go

```diff
@@ -0,0 +1,42 @@
+package virt_controller
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+)
+
+var (
+	vmSnapshotMetrics = []operatormetrics.Metric{
+		VmSnapshotSucceededTimestamp,
+	}
+
+	VmSnapshotSucceededTimestamp = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vmsnapshot_succeeded_timestamp_seconds",
+			Help: "Virtual Machine succeeded snapshots timestamp.",
+		},
+		[]string{"vm_name", "snapshot_name"},
+	)
+)
+
+func NewVMSnapshotCreated(snapshot *snapshotv1.VirtualMachineSnapshot) {
+	if snapshot.Status != nil &&
+		snapshot.Status.Phase == snapshotv1.Succeeded &&
+		snapshot.Status.CreationTime != nil {
+		VmSnapshotSucceededTimestamp.WithLabelValues(
+			snapshot.Spec.Source.Name,
+			snapshot.Name,
```

Comment:
this should be snapshot.Namespace?

Reply:
no I see you changed the metric to be vm name and snapshot name, you dont need namespace?

Reply:
This namespace that you see I dont think thats the namespace of the vm and vmsnapshot.. kubevirt is usually not the namespace you create those objects, usually your own namespace or the "default" one

Reply:
so you think we should add the snapshot.namespace as well?

Reply:
yes I think so.